lib/resque_bus/adapter.rb

```diff
@@ -11,16 +12,59 @@ def enabled!
         QueueBus::Worker.extend(::QueueBus::Adapters::Resque::RetryHandlers)
       end
 
-      def redis(&block)
+      def redis=(server)
+        @queuebus_redis = nil
+        @queuebus_redis_url = server
+      end
+
+      # This method returns the QueueBus Redis instance
+      # If it is not configured, it will default to the same instance as Resque.
+      # If it is configured, it will me intialized, memoized, and returned.
+      def queuebus_redis
+        @queuebus_redis ||
+          if @queuebus_redis_url
+            # The configuration has specified a custom server just for QueueBus.
+            # Resque doesn't offer a mechanism to create a Redis object without
+            # assignment, though, so we invoke assignment for its side-effects.
+
+            # Save the default Resque redis connection
+            default_redis = ::Resque.redis
```

Comment:
I was mostly concerned with putting `Resque.redis` back.
But maybe it's ok to raise as this is in a config situation and if that goes wrong, it's better to just freak out.